### PR TITLE
Made CSS Modular for MentorLogin.page.js

### PR DESCRIPTION
**Pull request by Sage-2001**
## Related Issue

Closes: #236 

## Description of Changes
- Created a separate CSS file named MentorLogin.css
- Redesigned the CSS and added some good properties.
- Kept the classes names meaningful

## Checklist:
- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots
Before
Desktop Screenshot 2023 06 05 - 10 38 53 32 (2)

After
Desktop Screenshot 2023 06 05 - 10 41 16 67 (2)

### CODE REVIEW

**Review by anmode — APPROVED** [review]
looks great. Could you improve the UI too? i

**Comment by anmode** [comment]
> > looks great. Could you improve the UI too? i
> 
> can you tell me what exactly needs to be improved on UI section

see user login page! I think may be similar

**Comment by Sage-2001** [comment]
> > > looks great. Could you improve the UI too? i
> > 
> > 
> > can you tell me what exactly needs to be improved on UI section
> 
> see user login page! I think may be similar

Desktop Screenshot 2023 06 06 - 10 44 53 33

Updated UI

**Comment by anmode** [comment]
> > > > looks great. Could you improve the UI too? i
> > > 
> > > 
> > > can you tell me what exactly needs to be improved on UI section
> > 
> > 
> > see user login page! I think may be similar
> 
> Desktop Screenshot 2023 06 06 - 10 44 53 33
> 
> Updated UI

feel's like You copied the Login Form. as `confirmPassword`? and other form fields. too please see!

**Comment by Sage-2001** [comment]
> > > > > looks great. Could you improve the UI too? i
> > > > 
> > > > 
> > > > can you tell me what exactly needs to be improved on UI section
> > > 
> > > 
> > > see user login page! I think may be similar
> > 
> > 
> > Desktop Screenshot 2023 06 06 - 10 44 53 33
> > Updated UI
> 
> feel's like You copied the Login Form. as `confirmPassword`? and other form fields. too please see!

Actually yes, I tried to copy the same functionalities 

> > > > > looks great. Could you improve the UI too? i
> > > > 
> > > > 
> > > > can you tell me what exactly needs to be improved on UI section
> > > 
> > > 
> > > see user login page! I think may be similar
> > 
> > 
> > Desktop Screenshot 2023 06 06 - 10 44 53 33
> > Updated UI
> 
> feel's like You copied the Login Form. as `confirmPassword`? and other form fields. too please see!

I understand your concerns about the previous design and styling of the prototype, and I apologize for any inconvenience it may have caused. I attempted to incorporate similar functionalities to avoid any awkwardness, but I now realize that it may not have aligned with the level of professionalism I strive for. I have since made significant changes and enhancements to the prototype, focusing on creating a more visually appealing and seamless user experience. I hope that these improvements address your expectations and demonstrate my commitment to delivering a high-quality product. Thank you for your understanding, and please let me know if you have any further feedback or suggestions.

Desktop Screenshot 2023 06 06 - 17 07 19 52

**Comment by anmode** [comment]
> > feel's like You copied the Login Form. as `confirmPassword`? and other form fields. too please see!
> 
> I understand your concerns about the previous design and styling of the prototype, and I apologize for any inconvenience it may have caused. I attempted to incorporate similar functionalities to avoid any awkwardness, but I now realize that it may not have aligned with the level of professionalism I strive for. I have since made significant changes and enhancements to the prototype, focusing on creating a more visually appealing and seamless user experience. I hope that these improvements address your expectations and demonstrate my commitment to delivering a high-quality product. Thank you for your understanding, and please let me know if you have any further feedback or suggestions.
> 
> Desktop Screenshot 2023 06 06 - 17 07 19 52

oh no worries! but please could you make changes to previous form? I mean it is ok you used login UI but just add fields confirm password and other option. ( as there is original mentor login).
if any query feel free to ask!!

**Comment by Sage-2001** [comment]
> > > feel's like You copied the Login Form. as `confirmPassword`? and other form fields. too please see!
> > 
> > 
> > I understand your concerns about the previous design and styling of the prototype, and I apologize for any inconvenience it may have caused. I attempted to incorporate similar functionalities to avoid any awkwardness, but I now realize that it may not have aligned with the level of professionalism I strive for. I have since made significant changes and enhancements to the prototype, focusing on creating a more visually appealing and seamless user experience. I hope that these improvements address your expectations and demonstrate my commitment to delivering a high-quality product. Thank you for your understanding, and please let me know if you have any further feedback or suggestions.
> > Desktop Screenshot 2023 06 06 - 17 07 19 52
> 
> oh no worries! but please could you make changes to previous form? I mean it is ok you used login UI but just add fields confirm password and other option. ( as there is original mentor login). if any query feel free to ask!!

@anmode what do you mean by previous UI? Do you mean the original mentor form? if yes then Every field that was present in the original login form is present in the current login form.

Original form 
https://user-images.githubusercontent.com/59249586/243257539-266aa285-7a0a-45e1-9020-fd1b069e97ba.png

New form 
https://user-images.githubusercontent.com/59249586/243682537-6c9cab71-b5bf-4f3e-9744-ea15f4651d46.png

If there is something that I missed please let me know, Thanks.

**Comment by Sage-2001** [comment]
> > > > > feel's like You copied the Login Form. as `confirmPassword`? and other form fields. too please see!
> > > > 
> > > > 
> > > > I understand your concerns about the previous design and styling of the prototype, and I apologize for any inconvenience it may have caused. I attempted to incorporate similar functionalities to avoid any awkwardness, but I now realize that it may not have aligned with the level of professionalism I strive for. I have since made significant changes and enhancements to the prototype, focusing on creating a more visually appealing and seamless user experience. I hope that these improvements address your expectations and demonstrate my commitment to delivering a high-quality product. Thank you for your understanding, and please let me know if you have any further feedback or suggestions.
> > > > Desktop Screenshot 2023 06 06 - 17 07 19 52
> > > 
> > > 
> > > oh no worries! but please could you make changes to previous form? I mean it is ok you used login UI but just add fields confirm password and other option. ( as there is original mentor login). if any query feel free to ask!!
> > 
> > 
> > @anmode what do you mean by previous UI? Do you mean the original mentor form? if yes then Every field that was present in the original login form is present in the current login form.
> > Original form https://user-images.githubusercontent.com/59249586/243257539-266aa285-7a0a-45e1-9020-fd1b069e97ba.png
> > New form https://user-images.githubusercontent.com/59249586/243682537-6c9cab71-b5bf-4f3e-9744-ea15f4651d46.png
> > If there is something that I missed please let me know, Thanks.
> 
> so the previous UI is this : https://user-images.githubusercontent.com/59249586/243552980-b102003f-381b-443a-90ce-5fa9a2396ec6.png as so use this and make sure to adjust it to have confirm passwors and other option which were in original form. I hope you get that

Done @anmode 
Desktop Screenshot 2023 06 07 - 21 26 33 99

**Review by anmode — CHANGES_REQUESTED** [review]
Cool, remove that suit png and use our png. that we have used in login page. 
@sayyedarib need your review lastly
LGTM

**Comment by Sage-2001** [comment]
> Cool, remove that suit png and use our png. that we have used in login page. @sayyedarib need your review lastly LGTM

Done @anmode 
Desktop Screenshot 2023 06 08 - 22 24 59 85

**Comment by Sage-2001** [comment]
> Change spelling of forget to forgot. And align it to left

The spelling is already forgot. Now I have aligned forgot password to the left
Desktop Screenshot 2023 06 09 - 09 19 01 82

**Review by sayyedarib — APPROVED** [review]
see the comment on line 91.

File: pages/mentorLogin.page.js [hunk 1]
```diff
@@ -81,73 +85,85 @@ function mentorLogin() {
...
-                />
-              </div>
-              {error && (
-                <div style={{ color: "red", gridColumn: "1/3" }}>{error}</div>
-              )}
-              <button
+      <main className="login-body">
+        <form
+          className="form-default"
+          action="login-bg.mp4"
```
[the remark this answers is not in the document]
Comment (Sage-2001) [reply pages/mentorLogin.page.js]: > see the comment on line 91. also fix formatting issue 😶

removed unnecessary stuff from the code + formatting using prettier is working fine on my local machine

**Comment by Sage-2001** [comment]
> see the comment on line 91. also fix formatting issue 😶

removed unnecessary stuff from the code + formatting using prettier is working fine on my local machine

**Comment by Sage-2001** [comment]
> > see the comment on line 91. also fix formatting issue 😶
> 
> removed unnecessary stuff from the code + Now formatting using prettier is working fine on my local machine

**Comment by Sage-2001** [comment]
> > > see the comment on line 91. also fix formatting issue 😶
> > 
> > 
> > removed unnecessary stuff from the code

**Comment by Sage-2001** [comment]
> > > > see the comment on line 91. also fix formatting issue 😶
> > > 
> > > 
> > > removed unnecessary stuff from the code



> > > > see the comment on line 91. also fix formatting issue 😶
> > > 
> > > 
> > > removed unnecessary stuff from the code + I tried running prettier locally on code it worked all right over there

**Review by sayyedarib — APPROVED** [review]

**Review by anmode — APPROVED** [review]
congrats @Sage-2001.... LGTM

**Comment by Sage-2001** [comment]
> congrats @Sage-2001.... LGTM

Thanks it was fun working with you @anmode and @sayyedarib

**Pull request merged**